vmm/src/memory_manager.rs

```diff
@@ -1023,9 +1023,10 @@ impl MemoryManager {
         #[cfg(feature = "tdx")]
         let dynamic = !tdx_enabled;
 
-        let hotplug_method = config.hotplug_method.clone();
-
-        let acpi_address = if dynamic && hotplug_method == HotplugMethod::Acpi {
+        let acpi_address = if dynamic
+            && config.hotplug_method == HotplugMethod::Acpi
+            && (config.hotplug_size.unwrap_or_default() > 0)
```

Comment:
Given a value of `0` is invalid to `hotplug_size` (according to our [document ](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/docs/memory.md#hotplug_size)and implementation [here](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/ec1679406e5f0e5e04b828dae9682164c61c9997/vmm/src/memory_manager.rs#L915-L918)), would it be better to check `config.hotplug_size.is_some()`?

Reply:
Yes that works as well. Either way, I'm okay with the PR, I'll let @rbradford decide!